examples/xyz.js

```diff
@@ -1,41 +1,15 @@
-goog.require('ol.Attribution');
 goog.require('ol.Map');
 goog.require('ol.View');
-goog.require('ol.control');
 goog.require('ol.layer.Tile');
-goog.require('ol.source.OSM');
 goog.require('ol.source.XYZ');
 
 
-var attribution = new ol.Attribution({
-  html: 'Tiles &copy; <a href="http://maps.nls.uk/townplans/glasgow_1.html">' +
-      'National Library of Scotland</a>'
-});
-
```

Comment:
Cool.  Sorry for implying that you were initiating a debate.  Your comments are great.  I was having flashbacks.